binderhub/templates/index.html

```diff
@@ -138,7 +138,7 @@ <h4 id="form-header" class='row'>Build and launch a repository</h4>
 
         <div id="build-progress" class="progress on-build hidden row">
           <div id="phase-failed" class="progress-bar progress-bar-danger progress-bar-striped hidden" style="width: 100%">
-            Failed
+            Failed. Please try refreshing this page.
```

Comment:
```suggestion
            Failed. try refreshing this page in a few seconds.
```
How about this? Was trying to think of a short sentence yet something that suggests that refreshing the page won't always fix things so that people aren't disappointed.

Reply:
Thanks! Looks great to me.